.env.test

```diff
@@ -10,12 +10,13 @@ GITHUB_PRIVATE_KEY=github_private_key
 GITHUB_APP_ID=github_app_id
 GITHUB_TOKEN=github_token
 GITHUB_WEBHOOK_SECRET=my_secret
-DATABASE_URL=mysql://root:akeneo_pim@127.0.0.1:3306/slub_test
+DATABASE_URL=mysql://slub:slub@mysql:3306/slub?serverVersion=5.7
```

Comment:
We may have an conflict between Docker and non Docker.
But I think Docker is really the way to go...